src/coreclr/jit/valuenum.cpp

```diff
@@ -83,6 +85,8 @@ struct DoubleTraits
         unsigned long long bits = 0xFFF8000000000000ull;
 #elif defined(TARGET_ARMARCH)
         unsigned long long bits = 0x7FF8000000000000ull;
+#elif defined(TARGET_LOONGARCH64)
+        unsigned long long bits = 0xFFF8000000000000ull;
```

Comment:
Looks the same as TARGET_XARCH; merge ifdefs

Reply:
This is a mistake. I will modify it.
Thanks